stable/phpbb/values.yaml

```diff
@@ -32,15 +36,49 @@ phpbbEmail: user@example.com
 # smtpUser:
 # smtpPassword:
 
+##
+## External database configuration
+##
+externalDatabase:
+  ## Database host
+  # host:
+
+  ## Database user
+  # user: bn_mediawiki
+
+  ## Database password
+  # password:
+
+  ## Database name
+  # database: bitnami_mediawiki
+
 ##
 ## MariaDB chart configuration
 ##
 mariadb:
+  ## Whether to use the database specified as a requirement or not. For example, to configure the chart with an existing database server.
+  enabled: true
+
   ## MariaDB admin password
   ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#setting-the-root-password-on-first-run
   ##
   # mariadbRootPassword:
 
+  ## Create a database
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#creating-a-database-on-first-run
+  ##
+  mariadbDatabase: bitnami_phpbb
+
+  ## Create a database user
+  ## ref: https://github.com/bitnami/bitnami-docker-mariadb/blob/master/README.md#creating-a-database-user-on-first-run
+  ##
+  mariadbUser: bn_phpbb
```

Comment:
i wonder if we should default this value to `root`

Reply:
sorry.. not here.
i meant default for externalDatabase.user

Reply:
I don't think so. We are not going to create a database but use an existing one. The applications should have their own user with restricted privileges in their database.